src/lualdap.c

```diff
@@ -822,9 +792,22 @@ static int get_attrs_param (lua_State *L, int idx, char *attrs[]) {
 		attrs[1] = NULL;
 	} else if (type != LUA_TTABLE)
 		attrs[0] = NULL;
-	else
-		if (table2strarray (L, lua_gettop (L), attrs, LUALDAP_MAX_ATTRS))
```

Comment:
What is the benefit of manually inlining the function? I would assume the compiler already does it automatically, and the function name and comment add a little bit of explanation.